index.html

```diff
@@ -21,6 +24,11 @@ <h1>Guilherme Lopes Martins | WDD 130</h1>
         </p>
         <p>Bye</p>
     </main>
+    <aside>
+        <h2>Brazil</h2>
+        <img src="../images/brasil.jpeg" alt="image of Brazil map">
```

Comment:
Since your image folder is in the same directory as the HTML file, you don’t need to use `../` to go one folder back. Remove it to make the image show up.

![image](https://github.com/user-attachments/assets/2e886a00-7972-45a8-9928-3f57ed7b6dfa)